tests/test_generators/input/kitchen_sink_compliance_inst_01.yaml

```diff
@@ -40,28 +37,25 @@
           - related_to: P:002
             type: NON_EXISTENT_RELATION
 - description: age lower than threshold
-  skip: false
   dataset:
     persons:
       - id: P
         name: benjamin button
         age_in_years: -5
 - description: age higher than threshold
-  skip: false
   dataset:
     persons:
       - id: P
         name: dr who
         age_in_years: 5000
 - description: additional properties is closed
-  skip: false
   dataset:
     persons:
       - id: P
         name: john smith
         made_up_property: "abc"
 - description: optional open properties
-  skip: true
+  skip: not_closed value does not propagate to nested objects. By design?
```

Comment:
hmm, perhaps Bill designed it that way, I can see the use case but a little unintuitive